Jenkinsfile_k8s

```diff
@@ -242,31 +242,20 @@ pipeline {
                   retry(count: 2, conditions: [kubernetesAgent(handleNonKubernetes: true), nonresumable()]) {
                     sh 'packer build -timestamp-ui -force -only="${PKR_VAR_image_type}.${PKR_VAR_agent_os_type}" ./'
                   }
-                  // si docker et buildtag alors :
-                  if (${compute_type} == 'docker' && ${TAG_NAME} != null) {
-                    echo "Pushing jenkinsciinfra/jenkins-agent-${agent_type}:${TAG_NAME} & jenkinsciinfra/jenkins-agent-${agent_type}:latest"
-                    infra.withDockerPushCredentials {
+                  // if docker and building a tag, push to dockerhub from inside the node
+                  // else we would loose the docker image
+                  if (compute_type == 'docker' && tag_name != null) {
+                    stage('Publish Docker image') {
```

Comment:
```suggestion
                    stage('Publish Docker images') {
```

Reply:
The singular maps to the fact that there is only 1 image pushed.
Tags are child concepts of a given image (and platform are grand-child).


Suggestions (but not mandatory and not strong):
- `stage("Publish Docker image ${agent_type} for ${pkr_var_architecture}")
- `stage('Publish all tags for Docker image')`